module/userProjects/helper.js

```diff
@@ -1474,7 +1474,9 @@ module.exports = class UserProjectsHelper {
 								projectCreation.data.userProfile = updatedUserProfile.data
 							}
 						}
-
+						if (bodyData.entityId !== '') {
```

Comment:
@Prajwal17Tunerlabs did we check test this api after this change? when we pass entityId in body data make sure it will not affect where ever we use the body data to determine the targeting of solution- IMPORTANT

Reply:
changes are made as per above mentioned scenario